retail-managment-system/rms-ui/src/app/pages/investor-page/investor-page.component.ts

```diff
@@ -98,6 +95,7 @@ export class InvestorPageComponent implements OnInit {
             duration: 2000,
           }
         );
+        this.dialog.open(AddTransactionComponent)
```

Comment:
use function you already added --->openTransactionDialog

Reply:
because in this function i want to send the object investor to get id but with the new investor i want just to open dialog